### ISSUE TITLE
Docs on error incorrect?

### ISSUE THREAD

**Issue by petermorlion** [lelylan/simple-oauth2#316]
I noticed that the error object that I receive is different from the docs.

The docs mention (under [errors](https://github.com/lelylan/simple-oauth2#errors)):

```
...
run();
// => {
//     "statusCode": 401,
//     "error": "Unauthorized",
//     "message": "invalid password"
// }
```

But the error I see is different, and matches the [Boom docs](https://hapi.dev/module/boom/api/?v=9.1.0). It has the following properties:

- data
- isBoom
- isServer
- output (which contains the statuscode, headers, etc)

So I suspect the docs of simple-oauth2 need to be updated?

**Comment by jonathansamines** [comment]
Yes, you are right, mind making a PR to update our docs?

**Comment by petermorlion** [comment]
Sure! Just wanted to check with you first. PR coming up.

**Comment by jonathansamines** [comment]
Closed by #317
